requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
```

Comment:
You need to run `make upgrade` with Python 3.8 other wise this could pull in package versions that are not compatible with 3.8 and break current deployments.

Reply:
I have rerun `make upgrade` with Python 3.8, please review the commit.